src/index.tsx

```diff
@@ -163,16 +240,28 @@ function App() {
             : DEFAULT_INITIAL_VIEW_STATE
         }
         controller={true}
-        layers={layers}
+        layers={
+          bboxSelectPolygonLayer
+            ? layers.concat(bboxSelectPolygonLayer)
+            : layers
+        }
         // @ts-expect-error
-        getTooltip={showTooltip && getTooltip}
+        getTooltip={showTooltip && isTooltipEnabled && getTooltip}
+        getCursor={() => (isDrawingBBoxSelection ? "crosshair" : "grab")}
         pickingRadius={pickingRadius}
+        onClick={onMapClickHandler}
+        onHover={onMapHoverHandler}
         useDevicePixels={isDefined(useDevicePixels) ? useDevicePixels : true}
         // https://deck.gl/docs/api-reference/core/deck#_typedarraymanagerprops
         _typedArrayManagerProps={{
           overAlloc: 1,
           poolSize: 0,
         }}
+        onLoad={() => {
+          actorRef.send({
+            type: "Deck.gl was loaded",
+          });
```

Comment:
What does this do?

Reply:
This is an action in the XState machine. It was added to signal that the Deck.gl component is ready, but in the current workflow it is not necessary and I removed it.